docs/source/conf.py

```diff
@@ -60,7 +60,7 @@
 
 # General substitutions.
 project = "traits"
-copyright = "2008-{date.year}, Enthought Inc".format(
+copyright = "2005-{date.year}, Enthought Inc".format(
```

Comment:
It would make more sense simply to hardcode this as "2005-2020", so that a simple text search and replace can be used when we want to update.

Reply:
Done.